test/unit/cookie.test.js

```diff
@@ -194,4 +194,26 @@ describe('Cookie', function () {
             })).valueOf()).to.eql('this is a cookie value');
         });
     });
+
+    describe('unparse', function () {
+        it('should return empty string if called with invalid cookies array', function () {
+            expect(Cookie.unparse('')).to.equal('');
```

Comment:
can we add assertions for other invalid types?

Reply:
Fixed.